packages/domain/generators/grouping-folder/create-app/src/lib/fe-app/add-dependencies.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  addDependenciesToPackageJson,
+  GeneratorCallback,
+  Tree,
+} from '@nrwl/devkit';
+import { FrontendFramework } from '../model/framework.enum';
+
+const frameworkDependencyMap = {
+  angular: {
+    '@jscutlery/cypress-angular': 'latest',
+    '@jscutlery/cypress-harness': 'latest',
+    '@angular/cdk': 'latest',
```

Comment:
I am planning at some stage to relook at how these kinds of dependencies are added. I think the preference should be to run the add generator and fall back to adding the dev dependency if the add generator doesn't exist.

Reply:
All good, this PR is just to get a dialogue going and present some ideas. For sure we should design it around a better structure and use more CLI parameters to instruct more generic/flexible generators and minimise duplication.

This PR is to have something to discuss around and get a general idea. We can always move things around and reduce duplication ;)